odoo/models.py

```diff
@@ -41,6 +41,7 @@
 import psycopg2
 from lxml import etree
 from lxml.builder import E
+from psycopg2.extensions import AsIs
```

Comment:
can not use sql.Identifier method of psycopg2.sql module   ?

Reply:
You are right, we should eventually use `psycopg2.sql.Identifier` for formatting table and column names. But that is not the point of this pull request. I would do a separate cleanup with such improvements in all places where we compose SQL queries programmatically.